src/inert.js

```diff
@@ -3,6 +3,14 @@
  * (http://www.w3.org/Consortium/Legal/2015/copyright-software-and-document).
  */
 
+// Fix issue in environments where Element is not defined
+Element = typeof Element === 'undefined' ? {
```

Comment:
Why use a ternary here, instead of an if statement?

Reply:
It's effectively equivalent to an if statement, generally I personally use a ternary for shorter if / else conditional statements, happy to change to an if statement if you think that'll make it more readable :)

Reply:
Yeah, since it goes over multiple lines and the "else" case is simply a no-op assignment, I think an if statement would be better here.